### ISSUE TITLE
ZZ Save session unexpected behaviour

### ISSUE THREAD

**Issue by Tritaya** [brookhong/Surfingkeys#1376]
## Error details

I have a few Chrome windows with taps:
win1 {tab1, tab2, tab3}
win2 {tab4, tab5, tab6}
win3 {tab7, tab8, tab9}
win4 {tab10, tab11, tab12}

I type ZZ to save the session.

The Chrome is closing.

I open the Chrome. It opens one of my windows (I have Chrome startup settings: Continue where you left off).
I type ZR and the session is restored like this:

win1 {tab1, tab2, tab3}
win2 {tab1, tab2, tab3}
win3 {tab1, tab2, tab3}
win4 {tab1, tab2, tab3}

In other words: multiple windows are restored with the SAME set of tabs instead of restoring windows with the respective individual set of the tabs.

SurfingKeys: 0.9.68

Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
